tests/Directory.Build.props

```diff
@@ -15,6 +15,12 @@
 
     <!-- This is useful for local runs -->
     <VSTestResultsDirectory>$(ArtifactsTestResultsDir)</VSTestResultsDirectory>
+
+    <!-- 
+      Test applications have their own way of referencing Aspire.Hosting.AppHost, as well as DCP and Dashboard, so we disable
+      the Aspire.Hosting.SDK targets that will automatically add these references to projects. 
+    -->
+    <SkipAddReferenceToDashboardAndDCP Condition="'$(TestsRunningOutsideOfRepo)' != 'true'">true</SkipAddReferenceToDashboardAndDCP>
```

Comment:
This should be moved to `tests/Shared/RepoTesting/Aspire.RepoTesting.props`, and can be dropped from `playground/Directory.Build.props`.